### ISSUE TITLE
Feature request: Please allow filenames to have .py extensions!

### ISSUE THREAD

**Issue by g40** [QQuick/Transcrypt#569]
Adding the filetype (on Windows at least) gives the following extremely counter-intuitive results. Also runs contrary to almost all of the existing documentation ...

Thanks for listening. Transcrypt is exceedingly useful.

```
>transcrypt -b -m -n wifi.py
Transcrypt (TM) Python to JavaScript Small Sane Subset Transpiler Version 3.7.4
Copyright (C) Geatec Engineering. License: Apache 2.0

Saving target code in: R:/src/wifi_server/wifi/__target__/org.transcrypt.__runtime__.js

Error while compiling (offending file last):
        File '<org.transcrypt.compiler.Module object at 0x0000019CA75EF4A8>', line 0, namely:

        Attempt to import module: wifi.py
        Can't find any of:
                R:/src/wifi_server/wifi/wifi/py.py
                R:/src/wifi_server/wifi/wifi/py.js
                r:/apps/python3/lib/site-packages/transcrypt/modules/wifi/py.py
                r:/apps/python3/lib/site-packages/transcrypt/modules/wifi/py.js
                R:/apps/python3/Scripts/wifi/py.py
                R:/apps/python3/Scripts/wifi/py.js
                r:/apps/python3/python37.zip/wifi/py.py
                r:/apps/python3/python37.zip/wifi/py.js
                r:/apps/python3/DLLs/wifi/py.py
                r:/apps/python3/DLLs/wifi/py.js
                r:/apps/python3/lib/wifi/py.py
                r:/apps/python3/lib/wifi/py.js
                r:/apps/python3/wifi/py.py
                r:/apps/python3/wifi/py.js
                r:/apps/python3/lib/site-packages/wifi/py.py
                r:/apps/python3/lib/site-packages/wifi/py.js
                r:/apps/python3/lib/site-packages/websocket_server-0.4-py3.7.egg/wifi/py.py
                r:/apps/python3/lib/site-packages/websocket_server-0.4-py3.7.egg/wifi/py.js
Aborted```

**Comment by JdeH** [comment]
Ok, it will be in the next release.
Thanks for the feedback!

**Comment by JdeH** [comment]
Fixed, committed, released as 3.7.6
